File: Python/BAEKJOON/BRONZE Ⅲ/5073 삼각형과 세 변.py
while True:
    A, B, C = map(int, input().split())
    max_length = max(A, B, C)
    is_invalid = (A + B + C) - max_length <= max_length

    if A == 0 and B == 0 and C == 0:
        break
    elif A == B and A == C and B == C:
        print("Equilateral")
    elif not is_invalid and (A== B or A == C or B == C):
        print("Isosceles")
    elif not is_invalid:
        print("Scalene")
    else:
        print("Invalid")

File: javascript/BAEKJOON/BRONZE Ⅲ/5073 삼각형과 세 변.js
input = require('fs').readFileSync(__dirname+'/test.txt').toString().split('\n');
//var input = require('fs').readFileSync('/dev/stdin').toString().split('\n');

for(let i = 0; i < input.length; i++) {
    const [A, B, C] = input[i].split(" ").map(Number);

    const maxLength = Math.max(A, B, C);
    const isInvalid = (A + B + C - maxLength) <= maxLength;

    if(A == 0 && B == 0 && C == 0) {
        break;
    }else if(A == B && A == C && B == C){
        console.log("Equilateral");
    }else if(!isInvalid && ((A == B) ||  (A == C) || (B == C))){
        console.log("Isosceles")
    }else if(!isInvalid){
        console.log("Scalene");
    }else{
        console.log("Invalid");
    }
}